x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/esql/discover_timeline_state_integration.cy.ts

```diff
@@ -104,13 +100,12 @@ describe.skip(
           req.alias = SAVED_SEARCH_UPDATE_WITH_DESCRIPTION;
         }
       });
-      login();
-      visitWithTimeRange(ALERTS_URL);
-      createNewTimeline();
-      goToEsqlTab();
-      updateDateRangeInLocalDatePickers(DISCOVER_CONTAINER, INITIAL_START_DATE, INITIAL_END_DATE);
-    });
-    context('save/restore', () => {
+    };
+
+    describe('save/restore', () => {
+      beforeEach(() => {
+        handleIntercepts();
+      });
       it('should be able create an empty timeline with default discover state', () => {
         addNameToTimelineAndSave('Timerange timeline');
         createNewTimeline();
```

Comment:
diving into this function, we should be able to remove the `should('be.visible')` [here](https://github.com/michaelolo24/kibana/blob/unskip-discover-state-tests/x-pack/test/security_solution_cypress/cypress/tasks/timeline.ts#L305) (Cypress [already checks](https://docs.cypress.io/guides/core-concepts/interacting-with-elements) for visibility)

Reply:
same thing like 123, looking into the `addDiscoverEsqlQuery` function, `should('be.visible')` from [here](https://github.com/michaelolo24/kibana/blob/unskip-discover-state-tests/x-pack/test/security_solution_cypress/cypress/tasks/discover.ts#L53)

Reply:
also, line 136, looking into the `openTimelineFromSettings` function, I think we should be able to delete [this line](https://github.com/michaelolo24/kibana/blob/unskip-discover-state-tests/x-pack/test/security_solution_cypress/cypress/tasks/timeline.ts#L388) entirely

Reply:
let's remove it from [here](https://github.com/michaelolo24/kibana/blob/unskip-discover-state-tests/x-pack/test/security_solution_cypress/cypress/tasks/timeline.ts#L408) as well, and also the `force: true`?

Reply:
Just gonna remove all of them in the timeline related files